docs/src/CommutativeAlgebra/localizations.md

```diff
@@ -184,21 +233,65 @@ MPolyLocalizedRingElem{
     MultSetType
   }
 ```
-In order to make the `Singular` functionality available there is 
+
+Ideals in localized polynomial rings are of type 
+```@docs
+MPolyLocalizedIdeal{BRT, BRET, RT, RET, MST}
+```
+Recall (see e.g. [Eis95]) that 
+if ``\mathbb k`` is a Noetherian ring, any localization ``W = R[U^{-1}]`` of a 
+multivariate polynomial ring ``R = \mathbb k[x_1,\dots,x_n]`` is again Noetherian and 
+any ideal ``I \subset W`` is of the form ``I = I'\cdot W`` for some ideal ``I' \subset R``. 
+This correspondence is not 1:1 but for any ideal ``I \subset W`` we always 
+have that 
+```math
+  J = \left\{ x\in R : \exists u \in U : u\cdot x \in I \right\}
+```
+is the unique element which is maximal among all ideals ``I'`` in ``R`` for 
+which ``I = I'\cdot W``. We call this the `saturated ideal` of the localization 
```

Comment:
Here.

Reply:
Ok, thx. I was looking in the *.jl files ony... Yes, this should be corrected.